edge-modules/api-proxy-module/docker/linux/amd64/Dockerfile

```diff
@@ -5,7 +5,7 @@
 
 ARG DIR=.
 
-FROM alpine:3.12.0
+FROM alpine:3.13.1
```

Comment:
We are still using 3.12 alpine in EdgeHub & EdgeAgent.

Reply:
Varun told me it is generating errors so I am updating it